src/extensions/action-handlers/zapier-action.js

```diff
@@ -71,9 +71,80 @@ export async function onTagUpdate(
 
 // What happens when a texter saves the answer that triggers the action
 // This is presumably the meat of the action
-export async function processAction() {
+export async function processAction({
+  questionResponse,
+  interactionStep,
+  campaignContactId,
+  contact,
+  campaign,
+  organization,
+  previousValue
+}) {
   try {
-    throw new Error("zapier-action.processAction is not implemented");
+
+    const baseUrl = getConfig("BASE_URL", organization);
+    const conversationLink = `${baseUrl}/app/${organization.id}/todos/review/${contact.id}`;
+    const payload = {
+      questionResponse: questionResponse,
+      interactionStep: interactionStep,
+      campaignContactId: campaignContactId,
+      contact: contact,
+      campaign: campaign,
```

Comment:
it will be rare, but there could be credential-y info in `campaign.features` object, so maybe in an abundance of caution let's do:
```
const campaignCopy = {...campaign};
delete campaignCopy.feature;
delete campaignCopy.features;
const payload = {
   questionResponse
   interactionStep,
   campaignContactId,
   contact,
   campaign: campaignCopy
....
   };
```
(note that in es6 you don't need to stutter the values that have the same name, so clean that up here, as well)